qemu/tests/numa_negative.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import re
+
+from virttest import error_context
+from virttest import virt_vm
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    numa options negative test:
+    1) Boot vm with incorrect numa options
+    2) Check if can get the expected qemu output
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    vm = env.get_vm(params["main_vm"])
+    params['start_vm'] = 'yes'
+    negative_type = params.get('negative_type')
+    error_msg = params.get('error_msg', '')
+    try:
+        vm.create(params=params)
+        output = vm.process.get_output()
+    except virt_vm.VMCreateError as e:
+        output = str(e)
+        if negative_type in ['threads_test', 'cpu_mismatch']:
```

Comment:
How about set a flag for these cases, in case we have more negative cases?

Reply:
Updated, use 'non-fatal' to represent it is just a warning, and qemu can launch successfully.
Please help review again, thanks!